content/patterns/feed/examples/js/feedDisplay.js

```diff
@@ -211,14 +211,31 @@ aria.FeedDisplay.prototype.renderItemData = function (itemData) {
     itemContent +=
       '<div class="restaurant-rating" id="' +
       ratingID +
-      '">' +
-      '<img class="restaurant-star-img" ' +
-      'alt="' +
-      itemData.rating +
-      ' stars" ' +
-      'src="imgs/rating-' +
+      '" data-rating-value="' +
       itemData.rating +
-      '.png">' +
+      '">' +
+      '<svg role="none" xmlns="http://www.w3.org/2000/svg" viewBox="38 -5 100 48" width="90" height="25">' +
+      '<defs>' +
+      '<g id="star">' +
+      '<polygon points="2.0,13.4 11.7,20.5 8.0,31.1 17.7,24.8 27.4,31.9 23.7,20.5 33.4,13.4 21.4,13.4 17.7,2.0 14.0,13.4" />' +
+      '</g>' +
+      '</defs>' +
+      '<g aria-label="one star" class="star-1 star-2 star-3 star-4 star-5">' +
```

Comment:
I don't know why, but this isn't working. When I look at the preview with JAWS in Chrome, the ratings graphics are not showing up.

Reply:
I presume the inline SVG in the radio-rating example *is* working? I'll see what I can figure out.

Reply:
Ah, I think I'm misunderstanding your comment, but just checked the VoiceOver output from the end-result code and that is also def. NOT working properly.

Reply:
Aria-label fixed in latest commit. Not 100% sure if that solves the issue you were having or not.

Reply:
Yes, you fixed it!!!

In this Tuesday's meeting, we'll work on getting reviewers assigned.